spec/factories/events.rb

```diff
@@ -123,6 +123,12 @@
       end
     end
 
+    trait :with_enough_free_places do
+      after(:build) do |event|
+        event.max_participants = event.application_letters.size
```

Comment:
Now the value gets set by hand in every test. I think that is currently the least ugly solution.